pom.xml

```diff
@@ -189,6 +189,47 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.revapi</groupId>
+          <artifactId>revapi-maven-plugin</artifactId>
```

Comment:
Actually a good point, but this is not 100% final config. I will move it to the parent from all other repositories once it is a final config. So if you do not mind I would leave it that way for now.